docs/data-clients/kubernetes.md

```diff
@@ -22,32 +21,35 @@ proxy made to apply updates very often. Skipper is used in
 production with more than 200.000 routing table entries.
 Skipper has Filters to change http data and Predicates to change the
 matching rules, both can combined and chained. You can set these in
-ingress.yaml files to build resiliency patterns like ratelimit or
+ingress.yaml files to build resiliency patterns like load shedding, ratelimit or
 circuitbreaker. You can also use them to build more highlevel
 deployment patterns, for example feature toggles, shadow traffic or
 blue-green deployments.
 
 Skipper's main features:
 
 - Filters - create, update, delete all kind of HTTP data
-  - [collection of base http manipulations](https://godoc.org/github.com/zalando/skipper/filters/builtin): for example manipulating Path, Querystring, ResponseHeader, RequestHeader and redirect handling
-  - [cookie handling](https://godoc.org/github.com/zalando/skipper/filters/cookie)
-  - [circuitbreakers](https://godoc.org/github.com/zalando/skipper/filters/circuit): consecutiveBreaker or rateBreaker
-  - [ratelimit](https://godoc.org/github.com/zalando/skipper/filters/ratelimit): based on client or backend data
-  - Shadow traffic: [tee()](https://godoc.org/github.com/zalando/skipper/filters/tee)
-- Predicates - advanced matching capability
-  - URL Path match: `Path("/foo")`
-  - Host header match: `Host("^www.example.org$")`
-  - [Querystring](https://godoc.org/github.com/zalando/skipper/predicates/query): `QueryParam("featureX")`
-  - [Cookie based](https://godoc.org/github.com/zalando/skipper/predicates/cookie): `Cookie("alpha", /^enabled$/)`
-  - [source whitelist](https://godoc.org/github.com/zalando/skipper/predicates/source): `Source("1.2.3.4/24")`
-  - [time based interval](https://godoc.org/github.com/zalando/skipper/predicates/interval)
-  - [traffic by percentage](https://godoc.org/github.com/zalando/skipper/predicates/traffic) supports also sticky sessions
+   - [collection of base http manipulations](../reference/filters.md):
+     for example [manipulating Path](../reference/filters.md#http-path), [Querystring](../reference/filters.md#http-query), [HTTP Headers](../reference/filters.md#http-headers) and [redirect](../reference/filters.md#http-redirect) handling
+   - [cookie handling](../reference/filters.md#cookie-handling)
+   - [circuitbreakers](../reference/filters.md#circuit-breakers)
+   - [ratelimit](../reference/filters.md#rate-limit): based on client or backend data
+   - [Shadow traffic filters](../reference/filters.md#shadow-traffic)
+- [Predicates](../reference/predicates.md) - advanced matching capability
+   - URL [Path](../reference/predicates.md#the-path-tree) match: `Path("/foo")`
+   - Host header match: `Host("^www.example.org$")`
```

Comment:
We can add link to the Host predicate as well.
https://opensource.zalando.com/skipper/reference/predicates/#host

Reply:
done